_opensearch/java-rest-high-level.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Java high-level REST client
+nav_order: 97
+---
+
+# Java high-level REST client
+
+The Elasticsearch OSS Java high-level REST client allows you to interact with your OpenSearch clusters and indices through request objects and response objects. To create an index, add data to a document, or complete some other operation, you submit request objects to your cluster, and you get back response objects that have all of the available information, such as the associated index or ID, returned from your cluster.
+
+## Setup
+
+To start using the Elasticsearch OSS Java high-level REST client, ensure that you have the following dependency in your project's `pom.xml` file:
+
+```
+<dependency>
+  <groupId>org.elasticsearch.client</groupId>
+  <artifactId>elasticsearch-rest-high-level-client</artifactId>
+  <version>7.10.2</version>
+</dependency>
+```
+
+You can now start your OpenSearch cluster. Any version of OpenSearch works with the Elasticsearch OSS high-level REST client.
+
+## Create high-level client
+
+Before you begin submitting request objects, you need to first create a high-level REST client object. The following example demonstrates how to establish a client that uses basic authorization credentials to access a cluster.
+
+If you're using the Elasticsearch OSS high-level REST client over HTTPS, you also need to point your java project to a keystore that has the appropriate security certificates.
+
+```java
```

Comment:
So we want a full, working, copy and pastable code sample. Explaining snippet by snippet is fine, but the entire thing needs to appear somewhere on the page. If your explanations are pretty quick, you can probably just leave comments in the sample about what you're doing.